R/read_data.R

```diff
@@ -20,7 +20,9 @@ read_csv_regions <- function(csv_file) {
 }
 
 read_csv <- function(...) {
-  utils::read.csv(..., stringsAsFactors = FALSE)
```

Comment:
Do we want to just strip *trailing* empty rows?  This removes internal empty rows too, which might be ok, but is more invasive.

Reply:
Yep intending to move any empty rows. I haven't seen any examples with empty rows in the middle of the data but I think this would not have meaning. Jeff is stripping these rows in naomi so I think this is the correct approach.